pkg/client/reputation.go

```diff
@@ -89,7 +89,7 @@ func (c *clientImpl) SendLocalTrust(ctx context.Context, prm SendLocalTrustPrm,
 type SendIntermediateTrustPrm struct {
 	iter uint32
 
-	trust *reputation.Trust
+	trust *reputation.PeerToPeerTrust
```

Comment:
Maybe it would be nice to name it `peerToPeerTrust` as well.

Reply:
There is no actual need since it's still trust, and the field is private